2019/pt-br/src/0xa7-security-misconfiguration.md

```diff
@@ -0,0 +1,80 @@
+API7:2019 Má Configuração de Segurança
+======================================
+
+| Agentes/Vetores | Fraquezas de Segurança | Impactos |
+| - | - | - |
+| Específico da API : Explorabilidade **3** | Prevalência **3** : Detecção **3** | Técnico **2** : Específico do negócio |
+| Atacantes eventualmente procuram falhas não corrigidas, *endpoints* comuns, ou diretórios não protegidos para ganhar acesso não-autorizado ou realizar um reconhecimento do sistema. | Configurações inadequadas de segurança podem ocorrer a qualquer nível do *stack* da API, desde o nível da rede até o nível da aplicação. Ferramentas de automação estão disponíveis para detectar e explorar erros de configuração, como serviços desnecessários e opções de suporte ao legado. | Configurações inadequadas podem não apenas expor dados sensíveis de usuários, como também podem revelar detalhes do sistema e comprometer servidores como um todo. |
+
+## A API está vulnerável?
+
+Sua API pode estar vulnerável se:
+
+* Configurações apropriadas de *hardening* faltam a qualquer parte da aplicação, ou quando há permissões mal configuradas em provedores de nuvem.
+* Os últimos *patches* de segurança não estão aplicados ou o sistemas estão desatualizados.
+* Recursos não necessários estão habilitados (ex.: certos verbos HTTP).
+* Transporte criptografado (TLS) não configurado.
+* Diretivas de segurança não enviadas aos clientes (ex.: [Cabeçalhos de Segurança][1]).
+* Configurações de política CORS (*Cross-Origin Resource Sharing*) não configuradas ou configuradas inadequadamente.
+* Mensagens de erro incluindo *stack trace* ou informações sensíveis.
+
+## Cenários de exemplo de ataques
+
+### Cenário #1
+
+Um atacante encontra o arquivo `.bash_history` no diretório root do servidor, o qual contém comandos utilizado pelo time de DevOps para acesso à API:
```

Comment:
fix
* where it's written "_(...) o qual contém comandos **utilizado** pelo (...)_"
* it should be "_(...) o qual contém comandos **utilizados** pelo  (...)_"